apps/wallet/src/ui/app/pages/home/transfer-coin/PreviewTransfer.tsx

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useMemo } from 'react';
+
+import { Text } from '_app/shared/text';
+import { IconTooltip } from '_app/shared/tooltip';
+import { TxnAddress } from '_components/receipt-card/TxnAddress';
+import { TxnAmount } from '_components/receipt-card/TxnAmount';
+import { parseAmount } from '_helpers';
+import { useAppSelector, useFormatCoin, useCoinDecimals } from '_hooks';
+import { GAS_SYMBOL, GAS_TYPE_ARG } from '_redux/slices/sui-objects/Coin';
+
+export type TransferCoinFormProps = {
+    coinType: string;
+    gasCostEstimation: number | null;
+    to: string;
+    amount: string;
```

Comment:
this should be bigint and always the not denominated value (like mist)

Reply:
this is a BigInt just passing input string see https://github.com/MystenLabs/sui/blob/d8d708d163a4540103736ba2570a2aed7080efdf/apps/wallet/src/ui/app/pages/home/transfer-coin/PreviewTransfer.tsx#L29

Reply:
lets change it to big int so there is no need to parse it again